indigo_api/views/documents.py

```diff
@@ -150,7 +152,33 @@ class AnnotationViewSet(DocumentResourceView, viewsets.ModelViewSet):
     permission_classes = DEFAULT_PERMS + (DjangoModelPermissionsOrAnonReadOnly, AnnotationPermissions)
 
     def filter_queryset(self, queryset):
-        return queryset.filter(document=self.document).all()
+        queryset = list(queryset.filter(document=self.document).all())
+
+        fake_annotations = []
+        for annotation in queryset:
+            if annotation.task and annotation.in_reply_to is None:
+                task_content_type = ContentType.objects.get_for_model(Task)
```

Comment:
Rather do this once, outside of the for loop -- it's not going to during iterations.